README.md

```diff
@@ -59,22 +59,18 @@ The following example shows a model combining linear and decision tree base lear
 model = lb.LBRegressor(base_models=(lb.models.Linear(), lb.models.Tree(max_depth=1),), **params).fit(X, y)
 ```
 
-<img src="examples/linear_model/linear_model.png" alt="drawing" width="800"/>
+<img src="_static/examples/linear_model.png" alt="drawing" width="800"/>
 
 The second example shows a model combining kernel ridge regression and decision tree base learners on the wine quality dataset.
 
 ```python
 model = lb.LBRegressor(base_models=(lb.models.KRR(sigma=0.5), lb.models.Tree(max_depth=5),), **params).fit(X, y)
 ```
 
-<img src="examples/kernel_ridge_regression/kernel_ridge_regression.png" alt="drawing" width="400"/>
+<img src="_static/examples/kernel_ridge_regression.png" alt="drawing" width="400"/>
 
 ## Installation
 
-Dependencies:
-- cunumeric
-- sklearn
```

Comment:
This list is out of date (omits `legate-boost`, `scipy`, and `numpy`). I think such lists in docs are hard to keep in sync with the project, and they don't provide much value beyond just directly inspecting `pyproject.toml`.

Proposing just removing this.

Reply:
Agreed